website/Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+cloudinary = "*"
+django = "*"
+django-author = "*"
+django-extensions = "*"
+django-filter = "*"
+djangorestframework = "*"
+dj-database-url = "*"
+drf-yasg = "*"
+invoke = "*"
+python-decouple = "*"
+psycopg2-binary = "*"
+django-storages = {version = "*", extras = ["google"]}
+
+[requires]
```

Comment:
@FaithDaka Let's remove this file out. We dont want to cause conflict with maintaining deployments. You can keep it locally in your environment incase you are using pipenv.

Reply:
I had added it to the .gitignore file in highlights branch. Let me pull changes and merge.